mlir/lib/Bindings/Python/IRCore.cpp

```diff
@@ -3238,6 +3238,25 @@ void mlir::python::populateIRCore(py::module &m) {
             return PyBlockArgumentList(self.getParentOperation(), self.get());
           },
           "Returns a list of block arguments.")
+      .def(
+          "add_argument",
+          [](PyBlock &self, const PyType &type, const PyLocation &loc) {
+            return mlirBlockAddArgument(self.get(), type, loc);
+          },
+          "Append an argument of the specified type to the block and returns "
+          "the newly added argument.")
+      .def(
+          "erase_argument",
+          [](PyBlock &self, unsigned index) {
+            return mlirBlockEraseArgument(self.get(), index);
+          },
+          "Erase the argument at 'index' and remove it from the argument list.")
```

Comment:
nit: This sounds like it actually erases the value but it doesn't right? It just removes from the argument list? If so, can you actually change the comment to say "but does not erase the value" or something like that.

Reply:
I can see you point and agree. I actually I borrowed the comment from the corresponding cpp APIs (https://github.com/llvm/llvm-project/blob/539b72f2e15f0d8a74a6c05c7085035040a3a831/mlir/include/mlir/IR/Block.h#L116). I believe the word  erase is used there to mean `arguments.erase(index)`, which also looks good to me.

Not sure if we update both and leave them as is.
wdyt?

Reply:
Nah in that case just leave both